taxonomy-editor-frontend/src/pages/go-to-project/index.tsx

```diff
@@ -1,66 +1,89 @@
 
-type ProjectType = {
-  id: string;
-  projectName: string;
-  taxonomyName: string;
-  ownerName: string;
-  branchName: string;
-  description: string;
-  errors_count: number;
-};
-
-export const GoToProject = () => {
-  const [projectData, setProjectData] = useState<ProjectType[]>([]);
+const ProjectsTable = ({ projects }: { projects: Project[] }) => {
   const navigate = useNavigate();
 
-  const { data, isPending, isError } = useFetch<ProjectsAPIResponse>(
-    `${API_URL}projects`
-  );
-
-  useEffect(() => {
-    let newProjects: ProjectType[] = [];
+  const columns: GridColDef<Project>[] = [
+    { headerName: "Project", field: "id", flex: 3 },
+    {
+      headerName: "Taxonomy",
+      field: "taxonomyName",
+      flex: 2,
+      valueFormatter: ({ value }) => toTitleCase(value),
+    },
+    { headerName: "Branch", field: "branchName", flex: 3 },
+    { headerName: "Owner", field: "ownerName", flex: 2 },
+    { headerName: "Description", field: "description", flex: 3 },
+    {
+      headerName: "Errors",
+      field: "errorsCount",
+      renderCell: ({ row }) => {
+        if (row.errorsCount == 0) {
+          return null;
+        }
 
-    if (data) {
-      const backendProjects = data.map(
-        ({
-          id,
-          branch_name,
-          taxonomy_name,
-          owner_name,
-          description,
-          errors_count,
-          status,
-        }) => {
-          return {
-            id, // needed by MaterialTable as key
-            projectName: id,
-            taxonomyName: toTitleCase(taxonomy_name),
-            ownerName: owner_name ? owner_name : "unknown",
-            branchName: branch_name,
-            description: description,
-            errors_count: errors_count,
-            status: status,
-          };
+        return (
+          <MuiLink
+            color="error"
+            href={`/${toSnakeCase(row.taxonomyName)}/${row.branchName}/errors`}
+            onClick={(event) => event.stopPropagation()}
+          >
+            {row.errorsCount + " errors"}
+          </MuiLink>
+        );
+      },
+    },
+    {
+      headerName: "Status",
+      field: "status",
+      renderCell: ({ row }) => {
+        if (row.status === ProjectStatus.EXPORTED) {
+          return (
+            <MuiLink
+              target="_blank"
+              rel="noopener"
+              href={row.githubPrUrl as string}
+              onClick={(event) => event.stopPropagation()}
+            >
+              {row.status}
+            </MuiLink>
+          );
         }
-      );
+      },
+    },
+  ];
 
-      newProjects = backendProjects;
-    }
+  const onRowClick = (params: GridRowParams<Project>) => {
+    navigate(
+      `/${toSnakeCase(params.row.taxonomyName)}/${params.row.branchName}/entry`
+    );
+  };
+
+  return (
+    <DataGrid
+      rows={projects}
+      columns={columns}
+      onRowClick={onRowClick}
+      pageSizeOptions={[25]}
```

Comment:
This does not work: I have 35 projects and they all show in the same page.
I made it work doing this:
```suggestion
      pageSizeOptions={[]}
      initialState={{
        pagination: { paginationModel: { pageSize: 25 } },
      }}
```